.coveragerc

```diff
@@ -17,3 +17,5 @@ exclude_lines =
     # Don't complain if tests don't hit defensive assertion code:
     raise AssertionError
     raise NotImplementedError
+# skip empty files, they seem to get misreported on codecov
+skip-empty = true
```

Comment:
This config line appears to have caused CodeCov to report missing coverage on `tests/__init__.py`. We either need to configure CodeCov to also skip empty files or remove this line and put up with empty files in our coverage reports.

Reply:
Here is coverage without skip-empty
```
@@            Coverage Diff             @@
##           master     #497      +/-   ##
==========================================
- Coverage   98.42%   98.34%   -0.08%     
==========================================
  Files          55       58       +3     
  Lines        2475     2606     +131     
  Branches      266      274       +8     
==========================================
+ Hits         2436     2563     +127     
- Misses         24       26       +2     
- Partials       15       17       +2     
```

The diff is 100% covered, I feel like the -0.08% is an error in  the change in coverage caldulation in this case.